src/inspect_ai/_view/www/src/client/api/api-view-server.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from "./types";
 
 /* global __VIEW_SERVER_API_URL__ */
-const API_BASE_URL = __VIEW_SERVER_API_URL__ || "";
+let API_BASE_URL = __VIEW_SERVER_API_URL__ || "";
```

Comment:
I don't really love using a global var here, open to better ideas

Reply:
Do you mean something like this? https://github.com/METR/inspect_ai/compare/react-lib...METR:inspect_ai:fetch-api-interface?expand=1

Reply:
Maybe better to do in a follow-on PR

Reply:
> Do you mean something like this? https://github.com/METR/inspect_ai/compare/react-lib...METR:inspect_ai:fetch-api-interface?expand=1

Yep, exactly. You could use your judgement about whether the `createViewServerApi` function is getting too long/ complex and refactor longer functions out of it (that get passed whatever state they need e.g. the fetchAPI or whatever) ....

I think this approach is much better than coordinating on that global...